src/geom/cell_hex8.C

```diff
@@ -368,4 +368,10 @@ Real Hex8::volume () const
     triple_product(q[2], q[4], q[5]) / 64.;
 }
 
+BoundingBox
+Hex8::loose_bounding_box () const
+{
+  return Elem::loose_bounding_box();
```

Comment:
I'm probably going to make myself look like an idiot here, but I just don't get it. `Elem::loose_bounding_box()` is already virtual and is not overridden by `Hex8` or any of its bases, so what is this actually doing differently from before?

Reply:
See Cell::loose_bounding_box()

Reply:
Ah, forgot about `Cell`. OK, now I get it :)